src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/abstractfactory/factory/factorycreators/ComponentFactoryCreator.java

```diff
@@ -0,0 +1,27 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.factory.factorycreators;
```

Comment:
Gdzieś tam dobrą praktyką jest używanie nazw pakietowych w liczbie pojedynczej, 
czyli 
```suggestion
package pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.factory.factorycreator;
```